app/src/components/v-form/form-field-raw-editor.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+	<v-dialog :model-value="showModal" persistent @esc="$emit('cancel')">
+		<v-card>
+			<v-card-title>{{ disabled ? t('view_raw_value') : t('edit_raw_value') }}</v-card-title>
+			<v-card-text>
+				<interface-input-code
```

Comment:
Great question. Once we start rendering more dedicated UI for different fields, it's really no longer a "raw" editor 🤔 But yeah, rendering multi-line for single-line field types feels a little odd too. @VincentKempers thoughts?

Reply:
On that note, there is also an existing `<system-raw-editor>` (which is used by Flows raw editor) that's another wrapper of codemirror, and does change between single/multi line depending on type. But there's also a bit of "hardcoded" stuffs such as using mustache language for $ prefixed variables syntax highlighting at the moment. 🤔 Not sure can we tweak it to be more agnostic.

Reply:
I agree it's kind of odd if you have to change single line fields. I could give crack at the `<system-raw-editor>`. We could also check if the type is an object and only use the `<interface-input-code>` for the object where the gutter and line number may be useful? the rest should be a textarea ?

Reply:
> We could also check if the type is an object and only use the `<interface-input-code>` for the object where the gutter and line number may be useful? the rest should be a textarea ?

That does sound good as well! Just not 100% sure would that feel inconsistent in terms of design (as in sometimes there's line number and gutter, sometimes there isn't). No qualms with it from a functionality perspective though 😄

Reply:
@azrikahar, I went for both`<system-raw-editor>` & `<interface-input-code>`. On fields like boolean it now shows single line and multiline for `'string'`. I do believe there should be some light validation on the object field because if I made a mistake in the raw I can't just see where it is therefore i can't add the json because of validation. Now it will show the input code without gutter and numbered lines but show a red line where you messed up.

Reply:
Sounds good! The language prop for system-raw-editor makes sense, as well as validation point. Will give it a spin to see how that goes 👍